x-pack/plugin/vectors/src/main/java/org/elasticsearch/xpack/vectors/mapper/DenseVectorFieldMapper.java

```diff
@@ -173,6 +172,11 @@ public DenseVectorFieldType fieldType() {
         return (DenseVectorFieldType) super.fieldType();
     }
 
+    @Override
+    public boolean parsesArrayValue() {
```

Comment:
Should this be final, like AbstractGeometryFieldMapper does?

Reply:
To me this could be a bit confusing, because no other methods on `DenseVectorFieldType` are final and it wasn't designed for extension.